Files gnome-pilot-2.0.10.orig/capplet/gnome-palm-conduit.png and gnome-pilot-2.0.10/capplet/gnome-palm-conduit.png differ
Files gnome-pilot-2.0.10.orig/capplet/gnome-palm.png and gnome-pilot-2.0.10/capplet/gnome-palm.png differ
diff -urN gnome-pilot-2.0.10.orig/capplet/gnome-pilot-capplet.c gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c
--- gnome-pilot-2.0.10.orig/capplet/gnome-pilot-capplet.c	2003-05-02 08:55:39.000000000 -0500
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c	2004-08-12 00:04:01.090633768 -0500
@@ -171,18 +171,18 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "CappletMain", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-capplet init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 	
 	if (!get_widgets (gpcap)) {
 		g_message ("gnome-pilot-capplet init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 	
 	fill_widgets (gpcap);
 	init_widgets (gpcap);
 
- error:
+ 
 }
 
 GnomePilotCapplet *
diff -urN gnome-pilot-2.0.10.orig/capplet/gnome-pilot-cdialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c
--- gnome-pilot-2.0.10.orig/capplet/gnome-pilot-cdialog.c	2003-04-22 15:51:06.000000000 -0500
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c	2004-08-12 00:04:01.096632856 -0500
@@ -112,17 +112,17 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "ConduitSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-cdialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpcd)) {
 		g_message ("gnome-pilot-cdialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 	
 	gnome_dialog_close_hides (GNOME_DIALOG (priv->dialog), TRUE);
 	
- error:
+ 
 }
 
 
diff -urN gnome-pilot-2.0.10.orig/capplet/gnome-pilot-ddialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c
--- gnome-pilot-2.0.10.orig/capplet/gnome-pilot-ddialog.c	2003-04-22 16:12:36.000000000 -0500
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c	2004-08-12 00:04:01.099632400 -0500
@@ -114,15 +114,15 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "DeviceSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-ddialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpdd)) {
 		g_message ("gnome-pilot-ddialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 
- error:
+ 
 }
 
 
diff -urN gnome-pilot-2.0.10.orig/capplet/gnome-pilot-druid.c gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c
--- gnome-pilot-2.0.10.orig/capplet/gnome-pilot-druid.c	2003-05-13 12:29:59.000000000 -0500
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c	2004-08-12 00:04:01.105631488 -0500
@@ -168,19 +168,19 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", NULL, NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-druid init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpd)) {
 		g_message ("gnome-pilot-druid init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 
 	map_widgets (gpd);
 	fill_widgets (gpd);
 	init_widgets (gpd);
 
- error:
+ 
 }
 
 
diff -urN gnome-pilot-2.0.10.orig/capplet/gnome-pilot-pdialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c
--- gnome-pilot-2.0.10.orig/capplet/gnome-pilot-pdialog.c	2003-04-23 10:39:44.000000000 -0500
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c	2004-08-12 00:04:01.112630424 -0500
@@ -135,15 +135,15 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "PilotSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-pdialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gppd)) {
 		g_message ("gnome-pilot-pdialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 
- error:
+ 
 }
 
 
Files gnome-pilot-2.0.10.orig/capplet/gnome-pilot-watermark.png and gnome-pilot-2.0.10/capplet/gnome-pilot-watermark.png differ